gcemetadata/lib/gcemetadata/gcemetadata.py

```diff
@@ -39,7 +39,7 @@ def __init__(self, apiv='v1'):
         self.server         = 'metadata.google.internal'
 
         if apiv not in self.getAvailableAPIVersions():
-            msg = 'Given API version "%s" not available' %s
+            msg = 'Given API version "%s" not available' %apiv
```

Comment:
pep8 clean up and renaming of all variables and functions/methods comes at the end, one ugly large pull request